### PR TITLE
Fix timer issue from #157 + test improvements

### PR DESCRIPTION
**Pull request by MagnusS**
- Fix timer issue from #157 by calling xmit in async
- Add pkt_drop profiling label when a packet is dropped in `Uniform_packet_loss` vnetif backend
- Remove 'input' debug message from segment.ml
- Enable debug logging in tests (from @talex5)
- Compile tests to native not bytecode (from @talex5)

### CODE REVIEW

File: tcp/segment.ml [hunk 1]
```diff
@@ -298,7 +297,7 @@ module Tx (Time:V1_LWT.TIME) (Clock:V1.CLOCK) = struct
                   Log.pf fmt "TCP retransmission on timer seq = %d"
                     (Sequence.to_int rexmit_seg.seq));
               (* FIXME: suspicious ignore *)
```
Comment (samoht) [inline tcp/segment.ml]: you can now remove that comment

File: tcp/segment.ml [hunk 2]
```diff
@@ -298,7 +297,7 @@ module Tx (Time:V1_LWT.TIME) (Clock:V1.CLOCK) = struct
...
-              xmit ~flags ~wnd ~options ~seq rexmit_seg.data >>= fun () ->
+              Lwt.async ( fun () -> xmit ~flags ~wnd ~options ~seq rexmit_seg.data );
```
Comment (samoht) [inline tcp/segment.ml]: you can also remove the spaces between the parentheses: 
```ocaml
Lwt.async (fun () -> xmit ~flags ~wnd ~options ~seq rexmit_seg.data);
```

**Comment by samoht** [comment]
can you rebase?

**Comment by MagnusS** [comment]
Rebased

File: _oasis [hunk 3]
```diff
@@ -291,13 +291,13 @@ Library "tcpip-stack-socket"
                       io-page.unix
 
 Executable test
+  CompiledObject:     native
```
Comment (samoht) [inline _oasis]: shoud be `Best` instead of `Native`

**Comment by samoht** [comment]
Merging, I'll do the last tweaks. Thanks all for finding and fixing the issue!

**Pull request merged**